src/Namespaced/NamespacedCachePool.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/*
+ * This file is part of php-cache organization.
+ *
+ * (c) 2015-2015 Aaron Scherer <aequasi@gmail.com>, Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Cache\Namespaced;
```

Comment:
Not really, unless you want to use names like sub pool or find a completely other term for it. As namespace is some what know for this, namespaced for the service sounds ok imo.